enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+---
+title: HyperShift Metrics Monitoring
+authors:
+- @bwplotka
+reviewers:
+- @jeremyeder
+- @smarterclayton
+- @kbsingh
+- @openshift/openshift-team-monitoring
+- @maorfr
+approvers:
+- .
+creation-date: 2021.12.08
+last-updated: 2021.12.08
+status: implementable
+---
+
+# HyperShift Metrics Monitoring
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Operational readiness criteria is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created.
+  - See [Documentation Required](#documentation-required) for further details.
+
+## Summary
+
+We propose to extract metric storage, alerting and querying to a separate service that can be scaled, deployed and managed in a decoupled mode. This will allow robust multi-cluster, fleet monitoring with best experience possible for cluster admins. This means SD org, running managed HyperShift will use RHOBS managed by Observability Group and AppSRE. 
+
+## Glossary
+
+* HSRE: HyperShift Admin SRE: People that will operate the Hypershift management cluster and its control planes. In practice, this means the SD org for ou managed solutions.
+* CUS: Customers. 
+* BU: Business Unit. 
+* MS-SRE/MT-SRE: Managed Service / Managed Tenant Site Reliability Engineer.
+* RHOBS: Red Hat Observability Service.
+* [Observatorium](http://observatorium.io/): Project for scalable SaaS based monitoring and observability.
+
+For additional acronyms, see [Red Hat Dictionary](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki)
+
+## Background
+
```

Comment:
nit: This domain/url is subject to change, let's reference the source repo https://github.com/openshift/hypershift which already contains the link to the docs.